DiffusionPreprocessing/scripts/run_eddy.sh

```diff
@@ -599,6 +599,8 @@ main()
 	eddy_command+="${sep_offs_moveOption} "
 	eddy_command+="${rmsOption} "
 	eddy_command+="${ff_valOption} "
+	eddy_command+="--cnr_maps "
+	eddy_command+="--residuals "
```

Comment:
Not sure if we should "hard code" these flags in -- that is kind of what we included the `--extra-eddy-arg` flag for.  I'm amenable to including `--cnr_maps` as a hard-coded default, since that seems relatively harmless, but I don't think we should hard-code in `--residuals`.

Reply:
`--residuals` does indeed produce a large output file. I can see the argument for leaving it as an option. We could also add a `--qc_options` flag to run_eddy.sh, which would add the flags needed to produce a full QC report (just `--cnr_maps` and `--residuals` in this case).

Reply:
Personally, I'd be fine with just hard coding in `--cnr_maps` as a flag.  One could argue that should always be generated anyway.  But leaving `--residuals` for specification via `--extra-eddy-arg`

Reply:
Ok, I'll just remove the `--residuals` flag from this patch